Source/Core/Core/Core.cpp

```diff
@@ -146,7 +168,14 @@ bool IsRunningInCurrentThread()
 
 bool IsCPUThread()
 {
-	return (s_cpu_thread.joinable() ? (s_cpu_thread.get_id() == std::this_thread::get_id()) : !s_is_started);
+	#ifdef ThreadLocalStorage
```

Comment:
Coding style: don't indent #ifdef/#else/#endif, put them on the first column of the line:

    bool IsCPUThread()
    {
    #ifdef ...
        return ...
    #else
    #endif
    }